.gitignore

```diff
@@ -1,21 +1,24 @@
-/application.yml
```

Comment:
Did you intentionally overwrite `.gitignore`? There are some entries here that should not be deleted. For example, I am now seeing `.factorypath` files.